ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/apitesting/SandboxServerResource.scala

```diff
@@ -51,7 +51,9 @@ class SandboxServerResource(sandboxConfig: SandboxConfig) extends Resource[Platf
 
   override def close(): Unit = {
     channel.shutdownNow()
-    channel.awaitTermination(5L, TimeUnit.SECONDS)
+    if (!channel.awaitTermination(5L, TimeUnit.SECONDS)) {
+      throw new RuntimeException("Unable to shutdown the grpc channel to Sandbox on time")
+    }
     eventLoopGroup.shutdownGracefully().await(10L, TimeUnit.SECONDS)
```

Comment:
@gleber-da: In `RemoteServerResource` we also throw in case the `eventLoopGroup` shutdown. Should we do this here now as well?

Reply:
Disregard, I see this will be [handled](https://github.com/digital-asset/daml/pull/1255/files#diff-f142c70b8a78e6493a6198e8b7272b28) in #1255